package.json

```diff
@@ -7,5 +7,9 @@
   "license": "MIT",
   "workspaces": [
     "packages/*"
```

Comment:
If you are willing to use Yarn's workspaces, then you should declare 2 workspaces: `server` and `client`, instead of `packages/*`.

https://classic.yarnpkg.com/en/docs/workspaces/#toc-how-to-use-it